index.html

```diff
@@ -1,59 +1,60 @@
 <!DOCTYPE html>
 <html lang="en">
 
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Freefolio</title>
-    <link rel="shortcut icon" href="./clean//img/logo.svg" type="image/x-icon">
-    <script src="//cdn.tailwindcss.com"></script>
-</head>
+    <head>
```

Comment:
Is this 2 spaces or 4 spaces? Use 2 spaces for tabs please, thanks.

Reply:
Here is the preview of the outcome.
![Screenshot 2022-10-03 at 10-29-11 Freefolio](https://user-images.githubusercontent.com/47875009/193491848-3bb742af-d52c-4233-85f8-c94dc64b8ece.png)
I'll create a new PR for the 2 spaces for the tabs.

Reply:
Nice! Thank you!